packages/app/templates/ui-extensions/projects/web_pixel_extension/package.json.liquid

```diff
@@ -4,6 +4,6 @@
   "main": "dist/main.js",
   "license": "UNLICENSED",
   "dependencies": {
-    "@shopify/web-pixels-extension": "^0.1.1"
+    "@shopify/web-pixels-extension": "^0.4.14"
```

Comment:
It's definitely not a dumb question and I did wonder myself if it would be better to default to latest, but as Sophie's example denotes, it's meant for 3P devs and it's best to give them a stable dev env in the end.